services/object/src/backends/aws.js

```diff
@@ -136,6 +173,34 @@ class AwsBackend extends Backend {
     }
     return true;
   }
+
+  /**
+   * Get the tags for this object in JSON object format {k: v}.
+   */
+  _objectTags(object) {
+    return {
+      ...this.tags,
+      ProjectId: object.project_id,
+    };
+  }
+
+  /**
+   * Construct the "Tagging" argument containing the tags for this object, as included
+   * in the `x-amx-tagging` header.
+   */
+  objectTaggingHeader(object) {
+    return qs.stringify(this._objectTags(object));
```

Comment:
I'm not deeply bothered by adding a new dependency to our pile of dependencies but superagent or got can't do something like this?